libr/core/cbin.c

```diff
@@ -455,7 +456,11 @@ static void _print_strings(RCore *r, RList *list, int mode, int va) {
 			pj_kn (pj, "length", string->length);
 			pj_ks (pj, "section", section_name);
 			pj_ks (pj, "type", type_string);
-			pj_ks (pj, "string", string->string);
+			if (!strcmp(encoding, "none")) {
```

Comment:
imho strings dont need to be encoded, only `e` elements

Reply:
good point. we may want anyway a `pj_ke` as @trufae suggested, but I think a global configuration of the encoding at the PJ level is required anwyay.

Reply:
we can have a global because that shouldnt be racy, but i would prefer not to change the constructor

Reply:
why?? Globals are bad, changing the constructor is the best way to go to provide options afaik.

Reply:
Agree with @ret2libc. Also it is highly important to also be able to serialize with fixed local-only settings. For example you wouldn't want these settings to affect projects for example, which should always use specific settings.

Reply:
If you don't want to change the ctor, we can find an alternative solution, for example an arg-less pj constructor that uses some default settings.

Reply:
yeah of course we can provide both arg-less constructore and option-based constructor, but the constructor is where those things shall be configured imo, not at the caller site like how it is done in this pr.